app/js/scoreChart.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Function to update the chart. Due to how chart.js works, the only effective way to do
+ * this seems to be completely resetting the graph container every time.
+ */
+function updateChart() {
+    document.querySelector('.show-graph').innerHTML = '<canvas id="myChart"></canvas>';
+    drawChart();
+}
+
+/**
+ * script for graph using chart.js, links to canvas id=myChart in adminPage.html
+ */
+function drawChart() {
+    sendToChart().then((chartData)=>{
```

Comment:
space before and after the fat arrow please!

Reply:
Done